.gitignore

```diff
@@ -1,6 +1,9 @@
 .idea/
 
+vendor/
```

Comment:
https://stackoverflow.com/questions/51118691/should-i-add-go-dep-vendor-to-gitignore

Reply:
The question you've linked is about go dep. When  using modules it doesn't make any sense:
1. All libraries versions are fixed in `go.mod` and `go.sum`.
2. While using go modules vendor is not automatically updated. So this folder doesn't obviously contain code that is actually used.

As an example hugo framework also has vendor in gitignore.
https://github.com/gohugoio/hugo/blob/master/.gitignore#L13

Reply:
suppose hugo doesn't use vendor cause they use proxy server for modules
https://docs.gomods.io/